sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -233,60 +234,115 @@ def test_get_token_for_teams_user_from_managed_identity(self, communication_live
+        _, user_id = self.generate_teams_user_aad_token()
         with pytest.raises(Exception) as ex:
-            token_response = identity_client.get_token_for_teams_user("invalid")
+            token_response = identity_client.get_token_for_teams_user(self.expired_teams_token, self.m365_app_id, user_id)
         assert str(ex.value.status_code) == "401"
         assert ex.value.message is not None
-
+    
+    @parameterized.expand(
+        [
+            ("empty_app_id", ""),
+            ("invalid_app_id", "invalid")
+        ]
+    )    
+    def test_get_token_for_teams_user_with_invalid_app_id(self, _, invalid_app_id):
+        if(self.skip_get_token_for_teams_user_test()):
+            return
+        identity_client = CommunicationIdentityClient.from_connection_string(
+            self.connection_str,
+            http_logging_policy=get_http_logging_policy()
+        )
+        aad_token, user_id = self.generate_teams_user_aad_token()
+        with pytest.raises(Exception) as ex:
+            token_response = identity_client.get_token_for_teams_user(aad_token, invalid_app_id, user_id)
+        assert str(ex.value.status_code) == "400"
+        assert ex.value.message is not None
+    
     @CommunicationPreparer()
-    def test_get_token_for_teams_user_with_expired_token(self, communication_livetest_dynamic_connection_string):
+    def test_get_token_for_teams_user_with_wrong_app_id(self, communication_livetest_dynamic_connection_string):
         if(self.skip_get_token_for_teams_user_test()):
             return
         identity_client = CommunicationIdentityClient.from_connection_string(
             communication_livetest_dynamic_connection_string,
             http_logging_policy=get_http_logging_policy()
         )
+        aad_token, user_id = self.generate_teams_user_aad_token()
+        with pytest.raises(Exception) as ex:
+            token_response = identity_client.get_token_for_teams_user(aad_token, user_id, user_id)
+        assert str(ex.value.status_code) == "400"
+        assert ex.value.message is not None
         
+    @parameterized.expand(
+        [
+            ("empty_user_id", ""),
+            ("invalid_user_id", "invalid"),
+        ]
+    )    
+    def test_get_token_for_teams_user_with_invalid_user_id(self, _, invalid_user_id):
+        if(self.skip_get_token_for_teams_user_test()):
+            return
+        identity_client = CommunicationIdentityClient.from_connection_string(
+            self.connection_str,
+            http_logging_policy=get_http_logging_policy()
+        )
+        aad_token, _ = self.generate_teams_user_aad_token()
         with pytest.raises(Exception) as ex:
-            token_response = identity_client.get_token_for_teams_user(self.expired_teams_token)
-        assert str(ex.value.status_code) == "401"
+            token_response = identity_client.get_token_for_teams_user(aad_token, self.m365_app_id, invalid_user_id)
+        assert str(ex.value.status_code) == "400"
```

Comment:
the same as for other languages - I wonder if this was supposed to change.

Reply:
we have already answered [here](https://github.com/Azure/azure-sdk-for-net/pull/26924#discussion_r878068558)